### DISCUSSION TITLE
PS 8.1.x - Pack product can be created even though they're empty

### DISCUSSION THREAD

**Discussion by florine2623** [PrestaShop/PrestaShop#34868] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

I can create a pack product with just a title, a quantity, a price and sell it.
But no specific products are added to it.

<img width="1217" alt="Screenshot 2023-12-14 at 15 22 50" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/f0164971-d185-4976-960b-85d21204b5c8">
<img width="1064" alt="Screenshot 2023-12-14 at 15 23 16" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/2a6a976e-ee48-4199-a3cb-99013dcd710b">

Found by @SD1982 

### Expected behavior

I should have an error when I try to create a Pack product without product.

```
Unable to update settings
This pack is empty. You must add at least one product item.
```

### Steps to reproduce

1. Go to BO > Product > Click Add new products > Choose pack of products > Enter a name > Add quantity > Add price
2. Click on save
3. See Successful update :x: 
I should have messages those messages displayed : 
"Unable to update settings"
"This pack is empty. You must add at least one product item."

### PrestaShop version(s) where the bug happened

8.1.x, develop

### PHP version(s) where the bug happened

8.1

**Comment by MatShir** [comment]
Converting it as a discussion. Like virtual products and combinations can be sold without the virtual file or combination. Should the user be warned that they are selling without them, or should the online publication be blocked?

**Reply by jf-viguier** [reply]
Warm the user only. The products can be populated by a sql request, a module or an import I guess.
